### DISCUSSION TITLE
Mode 1 - number of laps

### DISCUSSION THREAD

**Discussion by realm174** [Courseplay/courseplay#7430] [Q&A] [answered]
Hello all, In mode 1, there is a maximum of 20 laps that can be done.  I use this to take harvest from the silo and bring it to the sell point when the price is right. I usually end up doing much more than 20 trips.  Is there a way to bypass that 20 trips limit? Thanks!

**Comment by Tensuko** [comment, marked as answer]
Not yet. Take a 2nd Tractor with Trailer or a bigger Trailer.

**Reply by realm174** [reply]
sounds good, thanks! Is it a planned feature tho?

**Reply by Tensuko** [reply]
I have Ideas to make it work in a different way, but that is not rly a priority, as we have some much more important things to work on.

**Reply by realm174** [reply]
Of course, no worries! :) Thanks for the hard work! Much appreciated!
